src/utils/conductorConfig.js

```diff
@@ -0,0 +1,22 @@
+// Note, this is not really a config file. It's just a way for the JS to access the nix config
+// import config from '/Users/robbiecarlton/holo/hp-admin5/conductor-config.toml'
+import config from 'utils/conductor-config.toml'
+
+export function getAgent (index = 0) {
+  if (index >= config.agents.length) throw new Error(`There are less than ${index - 1} agents in the config`)
+  return {
+    id: config.agents[index].public_address,
+    nickname: config.agents[index].name,
+    conductorId: config.agents[index].id
+  }
+}
+
+export function findInstanceForAgent (instanceId, agentIndex = 0) {
```

Comment:
@zo-el - this is the function we need for `holochainClient` that we currently read the config toml to make it work. If you can see a way to do this just using conductor calls, that would be awesome.